manager/scheduler/filter.go

```diff
@@ -169,7 +169,7 @@ func (f *PluginFilter) Check(n *NodeInfo) bool {
 		}
 	}
 
-	if f.t.Spec.LogDriver != nil && f.t.Spec.LogDriver.Name != "none" {
+	if f.t.Spec.LogDriver != nil && (f.t.Spec.LogDriver.Name != "none" || f.t.Spec.LogDriver.Name != "") {
```

Comment:
Please add a comment for this conditional

Reply:
Done!